app/front/scripts/directives/visualizations/pivottable/template.html

```diff
@@ -11,6 +11,13 @@
       cube="{{ params.packageId }}"
       state="state"
       format-value="formatValue"
+      max-value-limit="1998"
+      max-value-message="{{ [
+          '<strong>Oh snap!</strong>',
```

Comment:
can you add this string to the translation file as well (you might consider using a shorter string here, or split these to a few separate strings.